nmmo/core/realm.py

```diff
@@ -76,7 +77,13 @@ def reset(self, map_id: int = None):
     self.log_helper.reset()
     self.event_log.reset()
 
-    self.map.reset(map_id or np.random.randint(self.config.MAP_N) + 1)
+    self.tick = 0
+
+    if self._replay_helper is not None:
+      self._replay_helper.reset()
```

Comment:
replay_helper.reset() now captures the packet, but it's called before respawning here 

looks like old code?